src/main/java/org/checkerframework/specimin/UnsolvedClassOrInterface.java

```diff
@@ -46,7 +46,7 @@ public class UnsolvedClassOrInterface {
   private boolean isThrowable = false;
 
   /** This field records if the class is an interface */
-  private final boolean isAnInterface;
+  public boolean isAnInterface;
```

Comment:
As a general rule, fields should be at most one of `public` and non-`final`, if at all possible. So, this change makes me very nervous (as it makes this field both).